src/app/pages/OnDemandAudioPage/OnDemandAudioPage.jsx

```diff
@@ -49,20 +51,30 @@ const StyledGelWrapperGrid = styled(GelPageGrid)`
   @media (min-width: ${GEL_GROUP_4_SCREEN_WIDTH_MIN}) {
     padding-top: ${GEL_SPACING_TRPL};
   }
+  @media screen and (-ms-high-contrast: none), (-ms-high-contrast: active) {
```

Comment:
This looks like media queries that are targeting IE browsers, is that what this is for? If so it's a bit of a hacky method, and reiterating @andrewscfc [comment](https://github.com/bbc/simorgh/pull/9060#issuecomment-823204912) it's probably better to fix the underlying bug before trying to apply the media query method

Reply:
Thank you for the review Harvey. The grid fallbacks are not applying on this component for internet explorer. You are right, there must be a much larger underlying cause. Let me investigate that :)